library/ga.js

```diff
@@ -1,6 +1,16 @@
 import ReactGA from 'react-ga';
 
 const trackId = 'UA-126214096-1';
```

Comment:
we should use env variable for this ua id ... it should be explicit in our github repo